.github/workflows/deploy.yml

```diff
@@ -3,7 +3,7 @@ name: Deploy
 on:
   push:
     branches:
-      - master
+      - "**"
```

Comment:
Can be simplified to single line `on: push`

Reply:
done